terragrunt/aws/vpc/outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "aws_security_group_ids" {
+  value = aws_security_group.efs_access_sg.id
```

Comment:
Could you add `description` attributes to both of these outputs.  It can 100% be a copy/paste of the `description` you set on the EFS module's input variables.